Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -959,7 +960,7 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
         sandbox_enable_state_flag("EnableExperimentalSandbox", *auditToken);
 #if USE(APPLE_INTERNAL_SDK)
     uint64_t bootTime = mach_boottime_usec();
-    if (!(bootTime & 0x7)) {
+    if (!(bootTime & 0x7) || CocoaApplication::isWebkitTestRunner() || CocoaApplication::isTestWebKitAPI()) {
```

Comment:
There is a `isRunningTest` in DefaultWebBrowserChecks.h. Maybe we can use that?

Reply:
> There is a `isRunningTest` in DefaultWebBrowserChecks.h. Maybe we can use that?

Ah, that is a great point, will fix.

Thanks for reviewing!